YakimS/assignment1/Q1

```diff
@@ -0,0 +1,71 @@
+public class Assignment1Q1
+{
+
+    public static void main(String[] args)
+    {
+        String s1 = "si Apepl Apepl ppppo";
+        String s2 = "ppeAl ppopp Apepl is";
+        System.out.println(isSentenceAnagram(s1, s2,true));
+        System.out.println(s1 + "      " + s2);
+    }
+
+    static boolean isSentenceAnagram(String sentence1, String sentence2, boolean isCaseSensitive)
+    {
+        String[] sent1 = sentence1.split(" ");
```

Comment:
`sent1` and `sent2` do not really say much about what those variables are. What kind of data do they contain? What names would make it more obvious?

Reply:
Maybe "wordsOfSentence1", "wordsOfSentence2"?